### PR TITLE
Remove hoomd1 support

### LINKED ISSUE #7: Remove hoomd 1 scripts
With hoomd now at V3, I think we can remove the hoomd 1 related bits and keep support for hoomd 2 and eventually add it for hoomd 3

### PR DESCRIPTION
**Pull request by chrisjonesBSU**
This should remove any options and checks for version 1 of hoomd. It still checks for version 2 and errors out if it's not able to be imported.

Addresses #7

### CODE REVIEW

File: msibi/optimize.py [hunk 1]
```diff
@@ -120,15 +120,9 @@ def optimize(
...
-
                 HOOMD_VERSION = 2
             except ImportError:
-                try:
-                    import hoomd_script
-
-                    HOOMD_VERSION = 1
-                except ImportError:
-                    raise ImportError("Cannot import hoomd")
+                raise ImportError("Cannot import hoomd")
```
Comment (jennyfothergill) [inline msibi/optimize.py]: I think we can change this to 
```suggestion
        if engine == "hoomd":
            import hoomd
            HOOMD_VERSION = 2
        else:  # don't need a hoomd version if not using hoomd
            HOOMD_VERSION = None
```
We don't need the try except if we just raise the exception anyway.

**Review by jennyfothergill — APPROVED** [review]
LGTM!

File: msibi/optimize.py (hunk 1 above)
Comment (jennyfothergill) [reply msibi/optimize.py]: Also sorry, the "suggestion" feature on lines with addition/deletion is really wonky.

Comment (chrisjonesBSU) [reply msibi/optimize.py]: Ok, made these changes if you want to double check.

**Pull request merged**